templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -47,55 +48,42 @@ if [[ "$TEST" == "plugin-from-pypi" ]]; then
 fi
 
 cd ../pulp-openapi-generator
-
+{%- if plugin_name != 'pulpcore' %}
 ./generate.sh pulpcore python
 pip install ./pulpcore-client
-{%- if plugin_name != 'pulpcore' %}
-./generate.sh {{ plugin_snake }} python
-pip install ./{{ plugin_snake }}-client
+rm -rf ./pulpcore-client
+if [[ "$TEST" = 'bindings' ]]; then
+  ./generate.sh pulpcore ruby 0
+  cd pulpcore-client
+  gem build pulpcore_client.gemspec
+  gem install --both ./pulpcore_client-0.gem
+fi
 {%- endif %}
-{%- for item in additional_repos if item.bindings | default(true) %}
```

Comment:
I believe this is why pulp-ansible is breaking,
https://github.com/pulp/pulp_ansible/blob/master/template_config.yml#L4-L8

Reply:
Yeah, I fixed this now.